client/composeApp/src/commonMain/kotlin/screen/MyLogScreen.kt

```diff
@@ -52,25 +54,38 @@ fun MyLogScreen(
             BottomNavigationBar(navController = navController)
         }
     ){
-        Column (modifier = Modifier.padding(it).padding(16.dp)) {
+        Column (modifier = Modifier
+            .background(color = Color(0xFFF5F4A5))
```

Comment:
.background(color = Color(0xFFF5F4A5))
            .padding(it)
            .padding(16.dp)
            .fillMaxSize()
            .verticalScroll(rememberScrollState())